packages/valory/skills/abstract_round_abci/tests/test_base.py

```diff
@@ -673,8 +674,9 @@ def test_increment_round_count(self) -> None:
         self.db.increment_round_count()
         assert self.db.round_count == 0
 
-    @mock.patch(
-        "packages.valory.skills.abstract_round_abci.base.is_json_serializable",
+    @mock.patch.object(
+        abci_base,
+        "is_json_serializable",
```

Comment:
There was an issue with the `open-aea`'s test tools and the following would not work:

```python
@mock.patch(
    "packages.valory.skills.abstract_round_abci.base.is_json_serializable",
    return_value=False,
)
```